src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -2478,6 +2478,25 @@ uint32 ObjectMgr::AddCreData(uint32 entry, uint32 mapId, float x, float y, float
     return spawnId;
 }
 
```

Comment:
Looks like an ad hoc function to me, please consider if it is necessary to have it in the global object acessor class or not